python/py-opt_einsum/Portfile

```diff
@@ -31,7 +31,7 @@ checksums           rmd160  fc1df640e738bb17d28e2772cdf9a2e678fafba2 \
                     sha256  59f6475f77bbc37dcf7cd748519c0ec60722e91e63ca114e68821c0c54a46549 \
                     size    73951
 
-python.versions     37 38 39 310
+python.versions     38 39 310 311
```

Comment:
py37-tensorflow depends on this port

Reply:
There is no `py37-tensorflow` (and there shouldn't be):

https://github.com/macports/macports-ports/blob/c5900b5a45d8d122736bf5ebd1cadd6e8e5c1361/python/py-tensorflow/Portfile#L42

Reply:
Ah right, I looked at the webapp at ports.macports.org; that seems to display inaccurate information.